CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 [comment]: <> "NOTE! Ensure all images are added via the \[label\]\(link\) syntax!"
 
+## 2024-08-05
+| What's new |
+| :--------- |
+| We're excited to announce a fresh new look and feel to Studio! In efforts to align with our branding, we've introduced new colors, revamped some of our icons, and updated fonts for better readability. We will be making more incremental changes in the upcoming months. We hope you enjoy the new look and please let us know if you notice any visual bugs!
```

Comment:
This already reads well! 🙌, here's a suggestion to make the cadence a little less repetitive/more concise.

Also--can we give them a specific way they should reach out if they notice any bugs? E.g., should they contact support? Reach out to their Apollo contact? A particular email? Something else?

```suggestion

| We're excited to unveil Studio's new look and feel! We've introduced brand-aligned colors, revamped icons, and updated fonts for better readability. Keep an eye out for more incremental updates in the coming months. We hope you love the new design—please let us know if you spot any visual issues.
```